src/plugins/shared_ux/public/components/empty_state/no_data_views/no_data_views_component.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiButton, EuiCard } from '@elastic/eui';
+import { css } from '@emotion/react';
+import { NoResultsIllustration } from '../assets';
+
+interface NoDataViewsComponentProps {
+  onClick?: () => void;
+  canCreateNewDataView: boolean;
+}
+
+const addDataViewText = i18n.translate('sharedUX.noDataViewsPage.addDataViewText', {
+  defaultMessage: 'Add a Data View',
+});
+
+const noDataViewsText = i18n.translate('sharedUX.noDataViewsPage.noDataViewsText', {
+  defaultMessage: 'No Data Views',
+});
+
+const noDescriptionText = i18n.translate('sharedUX.noDataViewsPage.descriptionText', {
+  defaultMessage: 'You have data in Elasticsearch, but no data views',
+});
+
+export const NoDataViewsComponent = (props: NoDataViewsComponentProps) => {
```

Comment:
This is great @majagrubic!

We currently have a design/component for the `No Data View` style prompt here: https://github.com/elastic/kibana/blob/main/src/plugins/data_view_editor/public/components/empty_prompts/empty_index_pattern_prompt/empty_index_pattern_prompt.tsx

![Screen Shot 2022-02-14 at 10 34 39 AM](https://user-images.githubusercontent.com/549577/153894683-ad8efe20-e2be-4b30-ae9d-f16cbdbd0ade.png)


Can you re-use that in some capacity instead? I don't know if it's better to copy/paste or just move the whole component and fix the references.